model/src/plugin_handler.cpp

```diff
@@ -206,9 +206,9 @@ class Plugin {
     m_abi_version = metadata.target_version;
     m_major_version = ocpn::split(m_abi_version.c_str(), ".")[0];
     m_name = metadata.name;
-    wxLogDebug("Plugin: setting up, name: %s", m_name);
-    wxLogDebug("Plugin: init: abi: %s, abi_version: %s, major ver: %s", m_abi,
-               m_abi_version, m_major_version);
+    DEBUG_LOG << "Plugin: setting up, name: " << m_name;
```

Comment:
Based on your comments, it seems the preferred way to log is `DEBUG_LOG` and `MESSAGE_LOG` instead of `wxLogDebug` and `wxMessageLog`. I wasn't sure what is the preferred way as this file contains multiple ways to log.